src/sentry/utils/performance_issues/detectors/io_main_thread_detector.py

```diff
@@ -193,8 +192,8 @@ def _is_io_on_main_thread(self, span: Span) -> bool:
         data = span.get("data", {})
         if data is None:
             return False
-        _, fileext = os.path.splitext(data.get("file.path", ""))
-        if fileext in self.IGNORED_EXTENSIONS:
+        file_path = data.get("file.path", "").lower()
+        if any(file_path.endswith(ext) for ext in self.IGNORED_LIST):
```

Comment:
If we're doing full filenames instead of extensions, can we do something more strict than `endswith`? I looked in the codebase and found a util called `glob_match` in `src/sentry/utils/glob.py`. Can we use change the `IGNORED_LIST` to a glob format (e.g. `*.nib`, `*.plist`, `kblayout_iphone.dat`) and use the util like:
```
if any(glob_match(file_path, ignored_pattern, ignore_case=True))
```
and potentially any other options that would be relevant here? There are some tests in `tests/sentry/utils/test_glob.py` that describes it behaviour more